frontend/src/index.js

```diff
@@ -1,9 +1,6 @@
-import React from 'react'
-import ReactDOM from 'react-dom'
+import React from 'react';
+import ReactDOM from 'react-dom';
 
-import App from './main/app'
+import App from './main/app';
 
-ReactDOM.render(
-  <App />,
-  document.getElementById('app')
-)
+ReactDOM.createRoot(document.getElementById('app')).render(<App />);
```

Comment:
This is not working @Anshgrover23 , we need to implement like the original one and just fix the space / linting issues